### PR TITLE
Updating all build versions

### PR DESCRIPTION
**Pull request by alexbarbato**
I have not tested this fully yet, I just removed all the build numbers.

I'm going to try to test now to ensure that none of those build versions are required for any of the components to work.

### CODE REVIEW

File: install-components.md [hunk 1]
```diff
@@ -10,27 +10,41 @@ Before you install the packages, ensure that you have completed the prerequisite
...
-+ [Install Supply Chain Choreographer](#install-scc)
-+ [Install Out of the Box Templates](#install-ootb-templates)
-+ [Install Default Supply Chain](#install-ootb-supply-chain-basic)
-+ [Install Supply Chain Security Tools - Store](#install-scst-store)
-+ [Install Supply Chain Security Tools - Sign](#install-scst-sign)
-+ [Install Supply Chain Security Tools - Scan](#install-scst-scan)
-+ [Install API portal](#install-api-portal)
-+ [Install Services Toolkit](#install-services-toolkit)
-+ [Install Tekton](#install-tekton)
+- [Installing Individual Packages](#installing-individual-packages)
```
Comment (alexbarbato) [inline install-components.md:13]: Wait, what happened here... I didn't add this... debugging

**Comment by cppforlife** [comment]
note that not all packages dropped build.X

```
dkalinin@cloudshell:~ (cf-sandbox-dkalinin)$ kubectl get pkg -A
NAMESPACE     NAME                                                                PACKAGEMETADATA NAME                                  VERSION          AGE
tap-install   accelerator.apps.tanzu.vmware.com.0.4.0                             accelerator.apps.tanzu.vmware.com                     0.4.0            235h42m28s
tap-install   api-portal.tanzu.vmware.com.1.0.3                                   api-portal.tanzu.vmware.com                           1.0.3            235h42m28s
tap-install   appliveview.tanzu.vmware.com.0.3.0                                  appliveview.tanzu.vmware.com                          0.3.0            52m11s
tap-install   buildservice.tanzu.vmware.com.1.3.1                                 buildservice.tanzu.vmware.com                         1.3.1            235h42m28s
tap-install   cartographer.tanzu.vmware.com.0.0.7                                 cartographer.tanzu.vmware.com                         0.0.7            235h42m27s
tap-install   cnrs.tanzu.vmware.com.1.0.3                                         cnrs.tanzu.vmware.com                                 1.0.3            235h42m28s
tap-install   controller.conventions.apps.tanzu.vmware.com.0.4.2                  controller.conventions.apps.tanzu.vmware.com          0.4.2            235h42m27s
tap-install   controller.source.apps.tanzu.vmware.com.0.1.2                       controller.source.apps.tanzu.vmware.com               0.1.2            235h42m27s
tap-install   developer-conventions.tanzu.vmware.com.0.3.0                        developer-conventions.tanzu.vmware.com                0.3.0            70h13m51s
tap-install   grype.scanning.apps.tanzu.vmware.com.1.0.0-beta.2                   grype.scanning.apps.tanzu.vmware.com                  1.0.0-beta.2     235h42m28s
tap-install   image-policy-webhook.signing.run.tanzu.vmware.com.1.0.0-beta.1      image-policy-webhook.signing.run.tanzu.vmware.com     1.0.0-beta.1     235h42m28s
tap-install   learningcenter.tanzu.vmware.com.1.0.14-build.1                      learningcenter.tanzu.vmware.com                       1.0.14-build.1   70h13m51s
tap-install   ootb-supply-chain-basic.tanzu.vmware.com.0.3.0-build.5              ootb-supply-chain-basic.tanzu.vmware.com              0.3.0-build.5    52m10s
tap-install   ootb-supply-chain-testing-scanning.tanzu.vmware.com.0.3.0-build.5   ootb-supply-chain-testing-scanning.tanzu.vmware.com   0.3.0-build.5    52m10s
tap-install   ootb-supply-chain-testing.tanzu.vmware.com.0.3.0-build.5            ootb-supply-chain-testing.tanzu.vmware.com            0.3.0-build.5    52m10s
tap-install   ootb-templates.tanzu.vmware.com.0.3.0-build.5                       ootb-templates.tanzu.vmware.com                       0.3.0-build.5    52m11s
tap-install   scanning.apps.tanzu.vmware.com.1.0.0-beta.2                         scanning.apps.tanzu.vmware.com                        1.0.0-beta.2     235h42m28s
tap-install   scst-store.tanzu.vmware.com.1.0.0-beta.1                            scst-store.tanzu.vmware.com                           1.0.0-beta.1     235h42m28s
tap-install   service-bindings.labs.vmware.com.0.5.0                              service-bindings.labs.vmware.com                      0.5.0            235h42m27s
tap-install   services-toolkit.tanzu.vmware.com.0.4.0                             services-toolkit.tanzu.vmware.com                     0.4.0            70h13m51s
tap-install   spring-boot-conventions.tanzu.vmware.com.0.1.2                      spring-boot-conventions.tanzu.vmware.com              0.1.2            235h42m28s
tap-install   tap-gui.tanzu.vmware.com.0.3.0                                      tap-gui.tanzu.vmware.com                              0.3.0            70h13m50s
tap-install   tap.tanzu.vmware.com.0.3.0                                          tap.tanzu.vmware.com                                  0.3.0            52m11s
tap-install   workshops.learningcenter.tanzu.vmware.com.1.0.7-build.1             workshops.learningcenter.tanzu.vmware.com             1.0.7-build.1    70h13m50s
```

File: install-components.md (hunk 1 above)
Comment (alexbarbato) [reply install-components.md]: silly vscode formatting...

**Comment by alexbarbato** [comment]
Yeah per @cppforlife for life's comments and my own (please don't merge until we've tested and updated the docs fully)

**Comment by BarrAndrea** [comment]
@alexbarbato  Has this been tested yet? Is this ready to merge?

**Review by BarrAndrea — APPROVED** [review]
I have reviewed and am approving this PR.

**Pull request merged**